### ISSUE TITLE
Song Editor: Rename tracks with left click.

### ISSUE THREAD

**Issue by Gabrielxd195** [LMMS/lmms#4985]
This is mentioned in # 4978. I can rename the tracks with a right click quickly and this is fine; but I can also rename with two clicks with the left button.

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/26127693/58054654-3552b880-7b29-11e9-8cda-6eae57cf61e8.gif)

This last option is very annoying because when you open an instrument or an editor quickly, it is unintentionally renamed. I think they should remove the renaming tracks feature with double left click and only rename with the right click.

**Comment by BaraMGB** [comment]
double click for renaming is a common behaviour. The user is used to this.

**Comment by Gabrielxd195** [comment]
> double click for renaming is a common behaviour. The user is used to this.

That's right, but also when you open a track quickly with double left click, automatically activates the text box to rename, and this behavior is a bit annoying, I think this works better with the right click.

**Comment by BaraMGB** [comment]
https://github.com/LMMS/lmms/pull/3356

I just want to bring in this old discussion. We wanted to rework the track label button after the 1.2 release.

**Comment by Sawuare** [comment]
I agree with OP. #4343 Fixes this issue.

**Comment by Sawuare** [comment]
> The user is used to this.

@BaraMGB If so, OP wouldn't have opened this issue and I wouldn't have opened #4343.

Why would you double-click to rename when you could just right-click?

**Comment by Spekular** [comment]
Why would you right click when you could just double click? It's begging the question to assume that right click is "easier" than double click or vice versa. Preference, setup, or habit could sway people in either direction.

Arguments for double click: Double click can be accomplished with a single button mouse/setup. Right click is often awkward to use on laptops in my experience.

Arguments for RMB: Inexperienced or older computer users may find it difficult to double click at the correct speed to trigger a double click. For this reason you could argue that double click never be the only way to access a function.

**Comment by noahb01** [comment]
Arguments for RMB: opening/closing instrument windows quickly often triggers a rename instead of the correct action.
